legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -0,0 +1,167 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This provides the service for involuntary dissolution."""
+import pytz
+from datedelta import datedelta
+from sqlalchemy import exists, not_
+
+from legal_api.models import Batch, BatchProcessing, Business, Filing, db
+from legal_api.utils.datetime import datetime
+
+
+class InvoluntaryDissolutionService():
+    """Provides services to get information for involuntary dissolution."""
+
+    @staticmethod
+    def get_businesses_eligible_count():
+        """Return the number of businesses eligible for involuntary dissolution."""
+        eligible_types = [
+            Business.LegalTypes.COMP.value,
+            Business.LegalTypes.BC_ULC_COMPANY.value,
+            Business.LegalTypes.BC_CCC.value,
+            Business.LegalTypes.BCOMP.value,
+            Business.LegalTypes.CONTINUE_IN.value,
```

Comment:
You might need CBEN as well https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/business.py#L72. Please confirm

Reply:
Good catch! Confirmed with Dylan and added it.